system/wymeditor/plugins/image_upload/jquery.wymeditor.image_upload.js

```diff
@@ -45,12 +45,37 @@ WYMeditor.editor.prototype.image_upload = function() {
 				} else {
 					jQuery(".wym_src", doc).val(response.thumbUrl);
 					jQuery(".wym_alt", doc).val(response.original_filename);
+					jQuery(".wym_attribution", doc).val(response.attribution);
 				}
 				jQuery("form#image_upload_form .submit", doc).val(oldSubmitLabel);
 			}
 		})
 	};
 
+	d.submitHandler = function (wDialog) {
```

Comment:
Why is this submitHandler completely new? I would think that adding attribution would only need to add one attribute to an existing handler. Is there maybe a hidden extra change here?

Reply:
No extra change, AFAIK the submitHandler needs to be overwritten in order to add a new attribute. Of course I'm open for suggestions.